examples/chatmemberbot.py

```diff
@@ -68,7 +68,7 @@ def extract_status_change(
     return was_member, is_member
 
 
-def track_chats(update: Update, context: CallbackContext.DEFAULT_TYPE) -> None:
```

Comment:
`CC` is a generic class, so you'll proper type hints for everything that's not generic. to get proper type hinting for `context.bot` and `context.{user, chat, bot}_data`, the type inspection needs to know the type of those generics. otherwise, it's just `Any`. If we make `CC` non-generic again, then we get type checking problems with using custom types for `user/chat/bot_data` …